### PR TITLE
Team H - NLP for robotics

### PR DESCRIPTION
**Pull request by sudo-prakhar**

### CODE REVIEW

**Comment by smbryan** [comment]
We're just missing an update to `navigation.yml`

CC: @nevalsar

**Comment by nevalsar** [comment]
@sudo-prakhar you can take a look at #139 for an example of changes required in `navigation.yml` for a new page.

File: wiki/machine-learning/nlp_for_robotics.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
+---
```
Comment (nevalsar) [inline wiki/machine-learning/nlp_for_robotics.md:1]: @sudo-prakhar This is a new page and requires an addition to the `navigation.yml`. You can take a look at #139 for an example of the changes required.

**Review by nevalsar — CHANGES_REQUESTED** [review]

**Review by nevalsar — CHANGES_REQUESTED** [review]
@sudo-prakhar You have merged from the master branch instead of rebasing your PR branch off it, which will lead to a circular history instead of a linear one. Can you fix that? 

Please remove the last merge commit and rebase your branch off the latest master branch.

**Comment by smbryan** [comment]
@nevalsar, Circular Git Histories? What an urban legend!

Recommended Reading: https://wiki.emulab.net/wiki/everything-you-need-to-unlearn-about-git

Merging in the master branch is a valid method of resolving potential conflicts. I'm guessing that @sudo-prakhar is still getting comfortable with Git & GitHub workflows (since it was easier to close the previous PR and open a new one when I requested changes). It would be good to brush up on rebasing, but isn't required in this case.

**Comment by nevalsar** [comment]
@smbryan I understand the needless 'linear history' confusion, but as per my understanding `git blame` loses usefulness if changes are commited back and forth across branches. Also, I felt there is no need to have the last merge commit as part of this PR since there is no merge conflict to resolve and changes in this PR are completely standalone.   

I usually require PRs to my repos to be rebased where possible but I agree that can be relaxed.

**Pull request merged**